node/container.go

```diff
@@ -336,9 +350,39 @@ func (a *HostAgent) PauseContainer(serviceID string, instanceID int) error {
 
 	// pause the running container
 	a.setInstanceState(serviceID, instanceID, service.StatePausing)
-	if err := attachAndRun(ctrName, svc.Snapshot.Pause); err != nil {
-		logger.WithError(err).Debug("Could not pause running container")
-		return err
+
+	go func() {
+		cmd := ""
+		if svc.Snapshot.Pause != "" {
+			cmd = svc.Snapshot.Pause + fmt.Sprintf(" & echo $! > %s", pidFile)
```

Comment:
Can you confirm that this command will still return an error if the underlying quiesce.sh returns a non-zero exit code?